bugbug/models/duplicate.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction import DictVectorizer
+from sklearn.pipeline import Pipeline
+import shap
+from bugbug import bug_features, bugzilla, feature_cleanup
+import pandas as pd
+from sklearn.feature_extraction.text import CountVectorizer, TfidfTransformer, TfidfVectorizer
+import scipy
+from sklearn.metrics import accuracy_score
+import numpy as np
+from sklearn.model_selection import train_test_split, cross_validate
+from sklearn.svm import LinearSVC
+
+
+
+class DuplicateModel:
+    def __init__(self, lemmatization=False):
+        self.cross_validation_enabled = False
+        self.sampler = None
+
+        self.calculate_importance = True
+        self.pipeline = Pipeline([
+            ("tfidf", TfidfVectorizer(preprocessor=self.transform)),
+            ("clf", LinearSVC())
+        ])
+
+    def transform(self, X):
+        cleanup_functions = [
+            feature_cleanup.responses(),
+            feature_cleanup.hex(),
+            feature_cleanup.dll(),
+            feature_cleanup.fileref(),
+            feature_cleanup.url(),
+            feature_cleanup.synonyms(),
+            feature_cleanup.crash(),
+        ]
+        for cleanup_function in cleanup_functions:
+            X = cleanup_function(X)
+        return X
+
+    def get_text(self, bug):
+        return bug["summary"] + " ".join(comment["text"] for comment in bug["comments"])
```

Comment:
Try what happens to the accuracy if you don't consider all comments, but just the first (replace `bug["comments"]` with `bug["comments"][0:1]`).

Reply:
If we consider the entire comment, accuracy is 0.9278897136797455.
If we consider only the first comment, accuracy is 0.9734888653234358
(Both were obtained using count=2500)